locales/languages.json

```
@@ -1 +1 @@
-["en", "ru", "zh", "de", "es", "fr", "cs"]
+["en", "bl", "zh", "de", "es", "fr", "cs" "ru"]
```

The iso code for Belarusian is `be`, and can you move both ru and be to the same place in the list?

```suggestion
["en", "zh", "de", "es", "fr", "cs", "be", "ru"]
```